tracking/imu/imu.h

```diff
@@ -1,6 +1,13 @@
-#pragma once
+#ifndef IMU_H
+#define IMU_H
 
 #include <stdbool.h>
+#include <furi_hal.h>
+
+#include "imu_t.h"
+#include "bmi160.h"
+#include "lsm6ds3trc.h"
+#include "lsm6dso.h"
```

Comment:
Let's delete all four of these files. :)
Because `imu_t.h` is not used anywhere else, and the only thing that's used from the other three is the `extern struct imu_t` part, so we might as well just list them here (that would still take one line per driver) and delete the headers.